### ISSUE TITLE
Deprecation

### ISSUE THREAD

**Issue by wraithm** [bitnomial/network-bitcoin#6]
We should point people @GambolingPangolin's wonderful new library:

https://github.com/bitnomial/bitcoind-rpc

We should also put a deprecation warning out on hackage and point people to that new lib.

**Comment by GambolingPangolin** [comment]
Agreed.  We should wait until we release `bitcoind-rpc` 0.3, since the current version does not cover all the functionality of this library.
